pkg/api/server/register_healthcheck.go

```diff
@@ -8,6 +8,29 @@ import (
 )
 
 func (s *APIServer) registerHealthCheckHandlers(r *mux.Router) error {
-	r.Handle(VersionedPath("/libpod/containers/{name}/runhealthcheck"), s.APIHandler(libpod.RunHealthCheck)).Methods(http.MethodGet)
+	// swagger:operation GET /libpod/containers/{name:.*}/runhealthcheck libpod libpodRunHealthCheck
```

Comment:
`/libpod/containers/{name:.*}/runhealthcheck` should be `/libpod/containers/{name:.*}/healthcheck`.  The verb GET is good.